src/main/fc/cli.c

```diff
@@ -891,17 +896,25 @@ static void cliSerialPassthrough(char *cmdline)
 
         passThroughPort = openSerialPort(id, FUNCTION_NONE, NULL,
                                          baud, mode,
-                                         SERIAL_NOT_INVERTED);
+                                         options);
         if (!passThroughPort) {
             cliPrint("could not be opened.\r\n");
             return;
         }
-        cliPrintf("opened, baud = %d.\r\n", baud);
+        cliPrintf("opened, baud = %d%s%s.\r\n", baud, (options & SERIAL_BIDIR) ? ",hd" : "", (options & SERIAL_INVERTED) ? ",inv" : "");
     } else {
         passThroughPort = passThroughPortUsage->serialPort;
+
+        // If the user supplied options, it can't be changed.
+        if (options != SERIAL_NOT_INVERTED) {
```

Comment:
`if (option != 0) {` (SERIAL_NOT_INVERTED is syntactic sugar defined as zero, but there are other options)